src/components/transactions/contract-call-details.tsx

```diff
@@ -1,56 +1,52 @@
 import React from 'react';
 import { useTxState } from '@common/hooks/use-tx-state';
-import { Box, Text, Flex } from '@stacks/ui';
+import { Stack, color, StackProps } from '@stacks/ui';
 import { Divider } from '@components/divider';
-import { truncateMiddle } from '@stacks/ui-utils';
-import { deserializeCV, cvToString, ClarityType, getCVTypeString } from '@stacks/transactions';
-import { LoadingRectangle } from '@components/loading-rectangle';
-import { NonceRow } from './nonce-row';
+import { deserializeCV, cvToString, getCVTypeString } from '@stacks/transactions';
 import { AttachmentRow } from './attachment-row';
-import { Link } from '@components/link';
 import { useExplorerLink } from '@common/hooks/use-explorer-link';
+import { Caption, Title } from '@components/typography';
+import { ContractPreview } from '@components/transactions/contract-preview';
+import { RowItem } from '@components/transactions/row-item';
 
 interface ArgumentProps {
   arg: string;
   index: number;
 }
 
-const Argument: React.FC<ArgumentProps> = ({ arg, index }) => {
+const Argument: React.FC<ArgumentProps> = ({ arg, index, ...rest }) => {
```

Comment:
Can we rename, this isn't very descriptive. Argument of what?

Reply:
renamed to `FunctionArgumentRow`